app/models/merchant.rb

```diff
@@ -13,9 +13,10 @@ def top_5_customers
   end
 
   def items_not_shipped
-    self.items.joins(:invoice_items)
-    .where("status!= 2")
-    .select("items.*, invoice_items.invoice_id")
+    self.items.joins(:invoices)
+    .where("invoice_items.status= 1")
```

Comment:
Good call on changing the status to 1 to include only packaged items.

Reply:
maybe change method name to packaged_items?